db/migrations/20210107185914-project-update.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('Projects', 'createdAt', { defaultValue: new Date(), type: Sequelize.DATE });
+    await queryInterface.changeColumn('Projects', 'updatedAt', { defaultValue: new Date(), type: Sequelize.DATE });
+
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+    */
+  },
+
+  down: (queryInterface, Sequelize) => {
+    /*
+      Add reverting commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.dropTable('users');
```

Comment:
Please add code here that undoes the changes in the `up` method and verify that they work by undoing this migration (and then redo this migration to get you database up to date)